Application/EdFi.Ods.Api/ScheduledJobs/Configurators/SchedulerConfiguratorBase.cs

```diff
@@ -60,7 +58,7 @@ public string GetCronExpression(ScheduledJobSetting scheduledJobSetting)
             }
             
             _logger.Warn($"Invalid cron expression provided for scheduled job: {scheduledJobSetting.Name}");
```

Comment:
Just a question, should the logger also inform that it will be reverting to the default chron expression (every 30 minutes repeatedly) due to the invalid app setting?

Reply:
I agree, pushed a change to include the value it is defaulting to in the log message